lms/static/scripts/frontend_apps/components/LMSGrader.js

```diff
@@ -36,11 +36,11 @@ export default function LMSGrader({
         },
       });
       // let the parent component know the index changed
-      onChangeSelectedUser(students[currentStudentIndex].userid);
+      onChangeSelectedUser(students[currentStudentIndex]);
     } else {
       // clear focused user
       removeClientConfig(['focus']);
```

Comment:
Is there any need to update the client config here still? When trying to resolve an issue locally to do with the username => userid change I got confused because this component is now updating the config in two different ways, but AFAIK only the `onChangeSelectedUser` change actually makes a difference.

Reply:
Oh ... I guess this will still matter if the sidebar has not yet loaded? In that case I think it is a bit confusing that the logic of deciding _how_ to handle a changed user is partly handled in the parent component (to make an RPC call) but partly handled in this component (to update the static config). I suggest that both should be handled in the same place.

Reply:
^ Makes sense, I'll move it to the parent